src/token/components/TokenImage.tsx

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from 'react';
+import { TokenImageReact } from '../types';
+
+export function TokenImage({ src, size = 24 }: TokenImageReact) {
+  const styles = useMemo(() => {
+    return {
```

Comment:
@abcrane123 @kyhyco how we style is something we need to figure out soon.

Having React styling can be painful in terms of perf.